runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -443,7 +443,7 @@ else if (config.swapBoxTrap() && (option.equals("check") || option.equals("disma
 		{
 			swap("reset", option, target, true);
 		}
-		else if (config.swapBoxTrap() && option.equals("take"))
+		else if (config.layBoxTrap() && option.equals("take"))
```

Comment:
:-1: If `layBoxTrap` is set off, the old behavior of `swapBoxTrap` being enabled is not maintained.

Reply:
Will fix that. It was kinda sloppy and hastily made for my own needs, so I didnt care about testing 👎